libraries/AP_HAL_ChibiOS/hwdef/AEROFOX-H7/defaults.parm

```diff
@@ -0,0 +1,27 @@
+ARSPD_TYPE	15
+BATT_VOLT_PIN	19
+BATT_VOLT_MULT	21
+BATT_MONITOR	3
+CAN_P1_DRIVER	1
+
+SERIAL1_BAUD	115

```

Comment:
You should not set parameters that are already the default as it consumes flash space. Ideally put these in the hwdef

Reply:
Thank you for your advice